gc/base/SparseVirtualMemory.cpp

```diff
@@ -138,7 +146,10 @@ MM_SparseVirtualMemory::allocateSparseFreeEntryAndMapToHeapObject(void *proxyObj
 #endif /* defined(OSX) || defined(OMRZTPF)) */
 
 	if (NULL != sparseHeapAddr) {
-		_sparseDataPool->mapSparseDataPtrToHeapProxyObjectPtr(sparseHeapAddr, proxyObjPtr, adjustedSize);
+		/* while the allocate and commit will work with _pageSize aligned memory, the map will contain exact size of the object
+		   (so we can later verify the size on updates).
+		 */
```

Comment:
`*` is missing and avoid first person.
```suggestion
		/* While the allocate and commit will work with _pageSize aligned memory, the map will contain exact size of the object.
		 * The size will be verified on updates.
		 */
```